spec/controllers/spree/orders_controller_spec.rb

```diff
@@ -168,21 +168,18 @@
 
   describe "removing items from a completed order" do
     context "with shipping and transaction fees" do
-      let(:order) { create(:completed_order_with_fees, shipping_fee: shipping_fee, payment_fee: payment_fee) }
+      let(:distributor) { create(:distributor_enterprise, charges_sales_tax: true, allow_order_changes: true) }
+      let(:order) { create(:completed_order_with_fees, distributor: distributor, shipping_fee: shipping_fee, payment_fee: payment_fee) }
       let(:line_item1) { order.line_items.first }
       let(:line_item2) { order.line_items.second }
       let(:shipping_fee) { 3 }
       let(:payment_fee) { 5 }
       let(:item_num) { order.line_items.length }
       let(:expected_fees) { item_num * (shipping_fee + payment_fee) }
-      let(:params) { { order: { line_items_attributes: {
-        "0" => {id: line_item1.id, quantity: 1},
-        "1" => {id: line_item2.id, quantity: 0}
-      } } } }
 
       before do
-        Spree::Config.shipment_inc_vat = true
-        Spree::Config.shipping_tax_rate = 0.25
+        allow(Spree::Config).to receive(:shipment_inc_vat) { true }
+        allow(Spree::Config).to receive(:shipping_tax_rate) { 0.25 }
```

Comment:
this is awesome! no need to use around block and keep original setting and restore it afterwards...

Reply:
Yep, I started doing that myself and then I though about this.